src/assets/i18n/en.json

```diff
@@ -11,7 +11,11 @@
     },
     "ERROR": {
         "PATIENT_SEARCH": "An error has occured while searching for patient, plese try again later",
-        "PATIENT_TABLE_GET_PATIENTS": "An error has occured while trying to load patients into a table, plese try again later"
+        "PATIENT_TABLE_GET_PATIENTS": "An error has occured while trying to load patients into a table, plese try again later",
+        "USER_INPUT_DIALOG_CREATE_PATIENT": "An error occurred while adding a new pacient, try again later"
+    },
+    "SUCCESS": {
+        "USER_INPUT_DIALOG_SUCCESS": "Registration was successful"
```

Comment:
"SUCCESS": {
        "USER_INPUT_DIALOG_SUCCESS":

There is no need to SUCCESSpart in USER_INPUT_DIALOG_SUCCESS, it will be better name as USER_INPUT_DIALOG_CREATE_PATIENT

Reply:
what do you mean?

Reply:
please rename: USER_INPUT_DIALOG_SUCCESS -> USER_INPUT_DIALOG_CREATE_PATIENT

Reply:
done